server/portal/apps/workspace/api/views.py

```diff
@@ -125,27 +127,16 @@ def get(self, request, *args, **kwargs):
 
 @method_decorator(login_required, name='dispatch')
 class JobsView(BaseApiView):
-    def get(self, request, *args, **kwargs):
+    def get(self, request, operation=None):
```

Comment:
We could also key off of `query_string` instead of `operation` to hit the search endpoint for a smaller diff

Reply:
I think long term, an `operation` based approach is the way to go, though it may be preferable if the `operation` mapped to tapis commands. For example, I wonder if it may be confusing to have a `select` operation, when `select` means something else in the tapis sense. What are your thoughts?

Reply:
Yeah that does make sense. I tried to closely mimic the tapis commands. For example the `listing` operation uses the tapis command `getJobSearchList` and the `search` operation uses the command `getJobSearchListByPostSqlStr`. Currently the `select` operation is mapped to the `getJob` command. Do you have a suggestion to what this could be renamed to?

Reply:
I think `select` makes sense, esp out of context of tapis. The multi purposed definition of `select` in context is less ideal, but perhaps acceptable. An earlier idea I had was to map our operation names one to one with tapis operation names. But that would require us to send something wordy like `getJobSearchListByPostSqlStr` as the `operation` across to the backend from the client side